docker/vpp-vswitch/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# compile vpp-agent needs vpp libs
+FROM contiv/vpp-base-builder:b59bd659 as builder
```

Comment:
Will we be manually updating this SHA1?  Could this become an `ARG`?

Reply:
Technically we could, but that needs to create a build hook in order to let docker cloud know which is the right image version and pass-in as build-args, however, that hook dir impacts all images built in this repo, so I think for now we just keep it in the dockerfile, it's not supposed to be changed frequently anyway

Reply:
Maybe just add a note that if you change the sha1 fragment here, you also need to change it down below too